tools/gotestloghelper/gotestevent/gotestevent.go

```diff
@@ -415,16 +451,24 @@ func JsonTestOutputToStandard(te *GoTestEvent, c *TestLogModifierConfig) error {
 			packageCoveragePrefixRegexp.MatchString(te.Output) {
 			return nil
 		}
+
+		//
```

Comment:
remove?

Reply:
I must have clicked merge just before this showed up. Will clean it up in another PR.